src/main/java/uk/gov/hmcts/reform/professionalapi/controller/external/OrganisationExternalController.java

```diff
@@ -136,7 +134,7 @@
 
         boolean isPendingPbaRequired = false;
 
-        if (!isBlank(pbaStatus)) {
+        if (StringUtils.isNotEmpty(pbaStatus)) {
```

Comment:
please verify org.apache.logging.log4j.util.Strings.isBlank and StringUtils.isNotEmpty internally works same , else there will be issue

Reply:
it does not 
`<pre>
     * StringUtils.isNotBlank(null)      = false
     * StringUtils.isNotBlank("")        = false
     * StringUtils.isNotBlank(" ")       = false
     * StringUtils.isNotBlank("bob")     = true
     * StringUtils.isNotBlank("  bob  ") = true
     * </pre>`
     
     `<pre>
     * StringUtils.isNotEmpty(null)      = false
     * StringUtils.isNotEmpty("")        = false
     * StringUtils.isNotEmpty(" ")       = true
     * StringUtils.isNotEmpty("bob")     = true
     * StringUtils.isNotEmpty("  bob  ") = true
     * </pre>`

Reply:
@amitkumarshrivastava  please revert the change OR find similar method , We cant afford production bug.

Reply:
@shreedhar-hmcts reverted !!!